BridgeAppSDK/SBASubtaskStep.swift

```diff
@@ -55,7 +55,7 @@ public class SBASubtaskStep: ORKStep {
     
     func substepIdentifier(identifier: String) -> String? {
         guard let range = identifier.rangeOfString("\(self.subtask.identifier).") else { return nil }
-        let stepRange = Range(start: range.endIndex, end: identifier.endIndex)
+        let stepRange = range.endIndex ..< identifier.endIndex
```

Comment:
Good question. Adding a test. I followed the instructions in the deprecation warning (and google) and unit tests still pass, but it doesn't really make sense. Checking now and will either update or just include the test.